parseSettings.js

```diff
@@ -1,37 +1,33 @@
 var baseUrl = "localhost:8887/";
 
 
-function convertServerSettingsToUrl(serverSettings) {
-	if (serverSettings !== undefined && serverSettings !== null) {
-	
-		var protocol = 'http://';
-				
-		var domain = baseUrl;
-				
-		var queryString = '';
-		if (serverSettings && serverSettings.cloudSiteCode && serverSettings.cloudSiteCode.length > 0) {
-			queryString =  '?sitecode=' + serverSettings.cloudSiteCode;
+function convertServerSettingsToUrl(serverSettings, launchUrl) {
```

Comment:
This now includes the original launch URL (if there was one), so those parameters can be passed on to the WebGL app along with the server settings.